ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activityNumericAnswer.component.ts

```diff
@@ -15,8 +16,7 @@ import * as _ from 'underscore';
                [min]="block.min"
                [max]="block.max"
                [placeholder]="placeholder || block.placeholder"
-               [attr.data-ddp-test]="'answer:' + block.stableId"
-               (blur)="onBlur($event.target.value)">
```

Comment:
We are using reactive form control, so using blur here makes no sense. We get values by subscribing to the control.